src/main/java/com/google/firebase/FirebaseApp.java

```diff
@@ -332,6 +349,36 @@ private void checkNotDeleted() {
     checkState(!deleted.get(), "FirebaseApp was deleted %s", this);
   }
 
+  private ListeningScheduledExecutorService getExecutorService() {
```

Comment:
How often is this not called? Can we initialize the executor in the constructor?